glean-core/src/lib.rs

```diff
@@ -21,11 +21,15 @@ use std::convert::TryFrom;
 use std::fmt;
 use std::sync::atomic::{AtomicBool, Ordering};
 use std::sync::{Arc, Mutex};
+use std::thread;
+use std::time::Duration;
 
-use metrics::MetricsDisabledConfig;
+use crossbeam_channel::unbounded;
 use once_cell::sync::{Lazy, OnceCell};
 use uuid::Uuid;
 
+use metrics::MetricsDisabledConfig;
```

Comment:
rebase garbage?

Reply:
how dare you call our code garbage?

nah, just reordering `use` statements and separating into blocks: std, external crates, internal modules